inst/config/tendril.yaml

```
@@ -1,7 +1,10 @@
-label: Tendril Plot {Tendril}
+env: safetyGraphics
+label: Tendril Plot
 type: plot
 domain: multiple
 package: safetyCharts
 export: true
 workflow:
   main: tendril_chart
+links: 
+  tendril: https://CRAN.R-project.org/package=tendril 
```

```suggestion
  tendril: https://cran.r-project.org/package=Tendril
```
Not sure why CRAN is upset at the capitalization but this one works for me